### PR TITLE
Events mobile

### PR DESCRIPTION
**Pull request by dylanlcole**

### CODE REVIEW

File: src/app/components/homePage/shared/time-selector.js [hunk 1]
```diff
@@ -18,70 +18,75 @@ class TimeSelector extends React.Component {
       })
       return (
        <div className='time-picker'>
-         <h3>Submit All Availability Times:</h3>
+         <h3>Submit Time Slots For Available Times:</h3>
```
Comment (Riffmeister) [inline src/app/components/homePage/shared/time-selector.js]: @dylanlcole I don't agree with this message as opposed to the previous? Can we revert back please? Or do a different variation?

File: src/app/shared/header.js [hunk 2]
```diff
@@ -52,6 +52,11 @@ _handleClick(event){
 		browserHistory.push('/vms')
 	}
   }
+
```
Comment (Riffmeister) [inline src/app/shared/header.js:55]: @dylanlcole Was this for that button you added and then took away?

File: src/app/components/homePage/shared/eventPortal.less [hunk 3]
```diff
@@ -56,7 +60,7 @@
     display: flex;
     flex-direction: column;
     flex-grow: 1;
-    height: 100%;
+    height: 90%;
```
Comment (Riffmeister) [inline src/app/components/homePage/shared/eventPortal.less:63]: @dylanlcole Why did we do this?

File: src/app/components/homePage/shared/time-selector.js (hunk 1 above)
Comment (dylanlcole) [reply src/app/components/homePage/shared/time-selector.js]: got it

File: src/app/shared/header.js (hunk 2 above)
Comment (dylanlcole) [reply src/app/shared/header.js:55]: yes

File: src/app/components/homePage/shared/eventPortal.less (hunk 3 above)
Comment (dylanlcole) [reply src/app/components/homePage/shared/eventPortal.less:63]: to give a buffer on the page I believe

Comment (dylanlcole) [reply src/app/components/homePage/shared/eventPortal.less:63]: think to give buffer to page

Comment (Riffmeister) [reply src/app/components/homePage/shared/eventPortal.less:63]: Are we trying to not have it cover the whole page, what do you mean by buffer?

**Pull request merged**